types/shelljs/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/glob": "^7.0.0"
```

Comment:
unfortunately, this won't work. Even if `@types/glob` is installed, Typescript will ignore it if `glob` is installed.